src/workloads/query/TimeSeriesLastpoint.yml

```diff
@@ -53,369 +54,330 @@ LastpointQueryMetaAscTimeAsc: &LastpointQueryMetaAscTimeAsc
 
   # Lastpoint query with a sort and group with meta subfield ascending and time ascending, but no index.
   - Name: RunLastpointQueryWithMetaSubfieldAscendingTimeAscendingNoIndex
+    Type: CrudActor
+    Database: *db
+    Phases:
+      OnlyActiveInPhases:
+        Active: [22]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          *LastpointQueryMetaAscTimeAsc
+
+  - Name: CreateIndexForMetaAscTimeAsc
     Type: RunCommand
     Database: *db
     Phases:
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *LastpointQueryMetaAscTimeAsc
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
+      OnlyActiveInPhases:
+        Active: [23]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          LoadConfig:
+            Path: *phasePath
+            Key: CreateIndex
+            Parameters:
+              Collection: *coll
+              IndexName: *metaDescTimeDesc   # We need to flip the direction for the lastpoint query
+              IndexPattern: *sortPatternMetaDescTimeDesc
+
+  # QuiesceActor will run in phase 24.
 
   # Lastpoint query with a compound index with meta subfield ascending and time ascending.
   - Name: RunLastpointQueryWithMetaSubfieldAscendingTimeAscending
+    Type: CrudActor
+    Database: *db
+    Phases:
+      OnlyActiveInPhases:
+        Active: [25]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          *LastpointQueryMetaAscTimeAscRepeat100
+
+  - Name: DropIndexForMetaAscTimeAsc
     Type: RunCommand
     Database: *db
     Phases:
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - LoadConfig:
-          Path: *phasePath
-          Key: CreateIndex
-          Parameters:
-            Collection: *coll
-            IndexName: &metaAscTimeAsc "MetaSubfieldAscendingTimeAscending"
-            IndexPattern: *sortPatternMetaAscTimeAsc
-      - *Nop
-      - *LastpointQueryMetaAscTimeAsc
-      - LoadConfig:
-          Path: *phasePath
-          Key: DropIndex
-          Parameters:
-            Collection: *coll
-            IndexName: *metaAscTimeAsc
+      OnlyActiveInPhases:
+        Active: [26]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          LoadConfig:
+            Path: *phasePath
+            Key: DropIndex
+            Parameters:
+              Collection: *coll
+              IndexName: *metaDescTimeDesc
 
   - LoadConfig:
       Path: *phasePath
       Key: QuiesceActor
       Parameters:
-        MaxPhases: 23
+        MaxPhases: *MaxPhases
 
 AutoRun:
```

Comment:
We are deprecating support for standalone variants. 
Can you remove the AutoRun conditions for the standalone setups? 
This would also then match when the HighCardinality workload gets scheduled. 

Also should the branch conditional be "$gte: v7.0" like it is for the HighCardinality one?

Reply:
> Can you remove the AutoRun conditions for the standalone setups?

This is done in #1219.

> should the branch conditional be "$gte: v7.0" like it is for the HighCardinality one?

This is answered in #1219.